### PR TITLE
Nk hw aqa 3

### PR DESCRIPTION
**Pull request by natak8200**

### CODE REVIEW

File: src/main/java/hw_aqa_3/task1.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+import java.time.Duration;
+
+public class task1 {
```
Comment (AntBelyi) [inline src/main/java/hw_aqa_3/task1.java]: Class name starts from the small letter?

File: src/main/java/hw_aqa_3/task2.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+import java.time.Duration;
+
+public class task2 {
```
Comment (AntBelyi) [inline src/main/java/hw_aqa_3/task2.java]: Class name starts from the small letter?

File: src/main/java/hw_aqa_3/task3.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+import java.time.Duration;
+
+public class task3 {
```
Comment (AntBelyi) [inline src/main/java/hw_aqa_3/task3.java]: Class name starts from the small letter?

File: src/main/java/hw_aqa_3/task4.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+import java.time.Duration;
+
+public class task4 {
```
Comment (AntBelyi) [inline src/main/java/hw_aqa_3/task4.java]: Class name starts from the small letter?

**Comment by AntBelyi** [comment]
Your task isn't compiled
image

File: src/main/java/hw_aqa_3/task1.java [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(5));
+        driver.manage().window().maximize();
+        driver.get("https://rozetka.com.ua");
+
+        WebElement accountButton = driver.findElement(By.cssSelector("[href='#icon-user-simple']"));
+        accountButton.click();
+
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("[title='Apple']")));
+
+        WebElement AppleButton = driver.findElement(By.cssSelector("[title='Apple']"));
```
Comment (AntBelyi) [inline src/main/java/hw_aqa_3/task1.java]: Variable name should starts from the small letter

File: src/main/java/hw_aqa_3/task4.java [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+        WebElement inputField = driver.findElement(By.cssSelector("#searchText"));
+        inputField.sendKeys("12455784", Keys.ENTER);
+
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("[class='not-found-delivery ng-star-inserted']")));
+
+        WebElement resultTextElement = driver.findElement(By.cssSelector("[class='not-found-delivery ng-star-inserted']"));
+
+        String resultText = resultTextElement.getText();
+
+        Assert.assertFalse(resultText.contains("resultTextElement"));
```
Comment (AntBelyi) [inline src/main/java/hw_aqa_3/task4.java]: Assertion is bad. Please fix it to more logic.

File: src/main/java/hw_aqa_3/task1.java (hunk 1 above)
Comment (natak8200) [reply src/main/java/hw_aqa_3/task1.java]: fixed

File: src/main/java/hw_aqa_3/task2.java (hunk 2 above)
Comment (natak8200) [reply src/main/java/hw_aqa_3/task2.java]: fixed

File: src/main/java/hw_aqa_3/task3.java (hunk 3 above)
Comment (natak8200) [reply src/main/java/hw_aqa_3/task3.java]: fixed

File: src/main/java/hw_aqa_3/task4.java (hunk 4 above)
Comment (natak8200) [reply src/main/java/hw_aqa_3/task4.java]: fixed

File: src/main/java/hw_aqa_3/task1.java (hunk 5 above)
Comment (natak8200) [reply src/main/java/hw_aqa_3/task1.java]: fixed

File: src/main/java/hw_aqa_3/task4.java (hunk 6 above)
Comment (natak8200) [reply src/main/java/hw_aqa_3/task4.java]: fixed

Comment (AntBelyi) [reply src/main/java/hw_aqa_3/task4.java]: Not. What do yo verify here?

**Pull request merged**